SU-SWS/stanford_profile
# Created a config split for acsf modules

# READY FOR REVIEW

# Summary
- Move acsf modules to an acsf config split.

# Need Review By (Date)
- later time is fine.

# Urgency
- low

# Steps to Test
1. using https://github.com/SU-SWS/acsf-cardinalsites/pull/94 verify the modules are still enabled on acsf environments.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
